src/main/java/frc/robot/Constants.java

```diff
@@ -72,7 +72,22 @@ public static class TurretConstants {
     public static final double KMAXOUTPUT = 1;
     public static final double KMINOUTPUT = -1;
     public static final int MAX_TURRET_ANGLE_DEGREES = 110;
-    public static final double CONVERSION_FACTOR = 0.25; // for the revbot prototype turret
+    public static final double TURRET_MOTOR_TURRET_RATIO = 0.2;
```

Comment:
What is this number from? It is a gear ratio?

Reply:
is currently a miscopy from rev bot, need a number from CAD